packages/devextreme/js/__internal/ui/html_editor/m_html_editor.ts

```diff
@@ -550,6 +557,17 @@ class HtmlEditor extends Editor<Properties> {
     this._formDialog = new FormDialog(this, userOptions);
   }
 
+  _renderAiDialog() {
```

Comment:
```suggestion
  _renderAIDialog() {
```
Please check all another places